src/modules/75-cf/components/CreateFlagWizard/FlagElemAbout.tsx

```diff
@@ -24,54 +26,79 @@ import * as Yup from 'yup'
 import { useStrings } from 'framework/strings'
 import { illegalIdentifiers } from '@common/utils/StringUtils'
 import { useTelemetry } from '@common/hooks/useTelemetry'
+import { Tag } from 'services/cf'
+import { useFeatureFlags } from '@common/hooks/useFeatureFlag'
 import { Category, FeatureActions } from '@common/constants/TrackingConstants'
 import type { FlagWizardFormValues } from './FlagWizard'
 import css from './FlagElemAbout.module.scss'
 
-interface FlagElemAboutProps {
+export interface FlagElemAboutProps {
   goBackToTypeSelections: () => void
+  tags: Tag[]
+  tagsError?: unknown
```

Comment:
As with the other place, you might wanna consider defaulting this and making it optional.
```suggestion
  tags?: Tag[]
  tagsError?: unknown
```